config/200-clusterrole-namespaced.yaml

```diff
@@ -49,3 +49,27 @@ rules:
     resources: ["*"]
     verbs: ["*"]
 
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: knative-eventing-namespaced-edit
+  labels:
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+    eventing.knative.dev/release: devel
+rules:
+  - apiGroups: ["eventing.knative.dev", "messaging.knative.dev", "sources.eventing.knative.dev"]
+    resources: ["*"]
+    verbs: ["create", "update", "patch", "delete"]
```

Comment:
Don't we need to either include the read APIs here or have the next CR also aggregate into the edit CR?

From https://kubernetes.io/docs/reference/access-authn-authz/rbac/#user-facing-roles **edit** is described as "Allows read/write access to most objects in a namespace. It does not allow viewing or modifying roles or rolebindings." So I think we need some form of read.

Reply:
As pointed out by @nak3 in https://github.com/knative/serving/pull/5683#discussion_r331363895, this is done automatically. We don't need to do anything for the view permissions to be aggregated into the edit permissions.

Reply:
Added a commit message note & rebase+squashed the commits to this effect